ElectronNET.API/NativeTheme.cs

```diff
@@ -32,11 +33,114 @@ internal static NativeTheme Instance
         }
 
         /// <summary>
-        /// A `Boolean` for if the OS / Chromium currently has a dark mode enabled or is
-        /// being instructed to show a dark-style UI.If you want to modify this value you
-        /// should use `themeSource` below.
+        /// Checks if the new ThemeSource is valid.
+        /// </summary>
+        /// <param name="themeSource">The new ThemeSource to check.</param>
+        /// <returns>True, if is a valid ThemeSource.</returns>
+        internal bool IsValidThemeSource(string themeSource)
```

Comment:
Can you offer this to an enum? Is type-safe and guides the developer in the flow ...